### PR TITLE
adding new marker for regression tests

### PR DESCRIPTION
**Pull request by SvetlanaRedHat**
This pr is to make new marker for scale regression cases.

Signed-off-by: savetisy <savetisy@redhat.com>

### CODE REVIEW

**Review by Avilir — COMMENTED** [review]
i think you need to add it to pytets.ini file as well

**Comment by wusui** [comment]
I think Avi is correct, but it is pytest.ini

add "regression: regression marker" (or some such comment) to the end of marker =

I am not sure if it messes things up if it  is added somewhere else.

**Comment by mbukatov** [comment]
Add explanation of exact meaning of this marker into the docs. Explanation why we need this could be also included here in the PR. The regression is a general term, and without further clarification, it won't be as useful as it could be.

**Comment by vasukulkarni** [comment]
what is the purpose for this one?

**Comment by tiffanyn108** [comment]
> what is the purpose for this one?

This marker will be used to run scale regression tests.  We should name as "scale_regression" to make it more clear about the purpose of the marker.

**Comment by tiffanyn108** [comment]
@SvetlanaRedHat There should be more test cases which needs to add scale regression marker.  Please review the spreadsheet for more info.

**Comment by tiffanyn108** [comment]
Need to validate this PR which Jenkins run.

**Comment by vasukulkarni** [comment]
> > what is the purpose for this one?
> 
> This marker will be used to run scale regression tests. We should name as "scale_regression" to make it more clear about the purpose of the marker.

Just "scale" should be sufficient in that case

File: ocs_ci/framework/pytest_customization/marks.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
 flowtests = pytest.mark.flowtests
 performance = pytest.mark.performance
 scale = pytest.mark.scale
+regression = pytest.mark.regression
```
Comment (vasukulkarni) [inline ocs_ci/framework/pytest_customization/marks.py:70]: please rename this to just "scale" for clarity

**Comment by tiffanyn108** [comment]
> > > what is the purpose for this one?
> > 
> > 
> > This marker will be used to run scale regression tests. We should name as "scale_regression" to make it more clear about the purpose of the marker.
> 
> Just "scale" should be sufficient in that case

We already have scale label for all scale tests.  We need a subset of scale test to run as scale regression.

File: ocs_ci/framework/pytest_customization/marks.py (hunk 1 above)
Comment (vasukulkarni) [reply ocs_ci/framework/pytest_customization/marks.py:70]: > > > > what is the purpose for this one?
> > > 
> > > 
> > > This marker will be used to run scale regression tests. We should name as "scale_regression" to make it more clear about the purpose of the marker.
> > 
> > 
> > Just "scale" should be sufficient in that case
> 
> We already have scale label for all scale tests. We need a subset of scale test to run as scale regression.

I see that, I think we could use pytest filter to achieve what we need, I really dont see need for another marker here if its just one test?

**Comment by tiffanyn108** [comment]
> > > > what is the purpose for this one?
> > > 
> > > 
> > > This marker will be used to run scale regression tests. We should name as "scale_regression" to make it more clear about the purpose of the marker.
> > 
> > 
> > Just "scale" should be sufficient in that case
> 
> We already have scale label for all scale tests. We need a subset of scale test to run as scale regression.

File: ocs_ci/framework/pytest_customization/marks.py (hunk 1 above)
Comment (SvetlanaRedHat) [reply ocs_ci/framework/pytest_customization/marks.py:70]: its not just one test, i added small group for test for fast jenkins run, adding the rest of  tests

**Comment by SvetlanaRedHat** [comment]
> > what is the purpose for this one?
> 
> This marker will be used to run scale regression tests. We should name as "scale_regression" to make it more clear about the purpose of the marker.

Done

**Comment by SvetlanaRedHat** [comment]
@tiffanyn108  updated to scale_regression
@Avilir  thanks a lot for your comment, i missed that part in my PR

**Review by wusui — APPROVED** [review]
Looks good

**Review by Avilir — APPROVED** [review]
LGTM

**Comment by jimdevops19** [comment]
I think it will be great to have the pr-validation executed here with `scale_regression` test mark already, should we add this value to the dropdown **before** this gets merged?

**Comment by vasukulkarni** [comment]
> I think it will be great to have the pr-validation executed here with `scale_regression` test mark already, should we add this value to the dropdown **before** this gets merged?

We are running just one test with the mark, I think it can be directly used as textbox, We have scale as mark so this might lead to some confusion.

**Comment by keesturam** [comment]
Closing this PR. Ram is addressing a similar change here - https://github.com/red-hat-storage/ocs-ci/pull/4543

**Pull request closed without merging**